firmware/src/kb-rzusbstick/include/arch/avr/usb_descriptors.h

```diff
@@ -130,8 +132,22 @@
  ,Usb_unicode('B') \
  ,Usb_unicode('0') \
  ,Usb_unicode('0') \
- ,Usb_unicode('1') \
+ ,Usb_unicode('T') \
 }
+#else
+#define USB_PRODUCT_NAME \
+{ Usb_unicode('K') \
+ ,Usb_unicode('I') \
+ ,Usb_unicode('L') \
+ ,Usb_unicode('L') \
+ ,Usb_unicode('E') \
+ ,Usb_unicode('R') \
+ ,Usb_unicode('B') \
+ ,Usb_unicode('0') \
+ ,Usb_unicode('0') \
+ ,Usb_unicode('2') \

```

Comment:
Should version numbering match the firmware name, e.g. 006?

Reply:
Makes sense! I'll tweak...